src/sss_client/pam_sss_gss.c

```diff
@@ -32,6 +31,14 @@
 #include <unistd.h>
 #include <string.h>
 
+#include "config.h"
+
+#ifdef HAVE_SECURITY_PAM_EXT_H
+#include <security/pam_ext.h>
+#else
+#define pam_syslog(...)
```

Comment:
Not sure what is the difference `pam_syslog` vs `pam_vsyslog` and the latter could be used in this file.

If not then for consistency it's better to provide `pam_syslog` in compat header as well.